src/jit/gentree.cpp

```diff
@@ -6308,9 +6308,9 @@ GenTreePtr          Compiler::gtCloneExpr(GenTree * tree,
         }
         copy->gtCall.gtRetClsHnd = tree->gtCall.gtRetClsHnd;
 
-#if defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
-        copy->gtCall.structDesc.CopyFrom(tree->gtCall.structDesc);
-#endif // defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)  
+#ifdef FEATURE_UNIX_AMD64_STRUCT_PASSING
+        copy->gtCall.gtMultiRegReturnTypeDesc = tree->gtCall.gtMultiRegReturnTypeDesc;
+#endif
 
```

Comment:
If call node is returning a scalar type in a single register, gtType of GenTree node will fullly describe the return type. We would need a Return type desc only for multi-reg returns.